sqlglot/expressions.py

```diff
@@ -167,7 +175,23 @@ class CTE(Expression):
 
 class Column(Expression):
     token_type = TokenType.COLUMN
-    arg_types = {"this": True, "db": False, "table": False, "fields": False}
+    arg_types = {"this": False, "db": False, "table": False, "fields": False}
```

Comment:
Why is `this` no longer required?

Reply:
because if we set fields (a.b.c.d.e) i make it null, basically highly nested structs

Reply:
ah, that makes sense. I wonder whether we need some sort of a custom validation here. Like eg either this or that argument must be present.

Reply:
btw, parsing of deeply nested structs looks weird but doesn't fail with the current implementation:
```python
>>> from sqlglot import parse_one
>>> parse_one("a.b")
(COLUMN this: b, db: , table: a, fields: )
>>> parse_one("a.b.c")
(COLUMN this: c, db: a, table: b, fields: )
>>> parse_one("a.b.c.d")
(COLUMN this: c, db: a, table: b, fields: a, b, c, d)
>>> parse_one("a.b.c.d.e")
(COLUMN this: c, db: a, table: b, fields: a, b, c, d, e)
>>> parse_one("a.b.c.d.e.f")
(COLUMN this: c, db: a, table: b, fields: a, b, c, d, e, f)
```